src/main/resources/screens/live/COMP_SERVICE_TRIAGE_TRANSFER.json

```diff
@@ -10,23 +10,7 @@
       "props": {},
       "component": "text-area",
       "name": "CaseNote_TriageTransfer",
-      "label": "Enter reason for transfer"
-    },
-    {
```

Comment:
How does this impact old cases on the old version of the workflow that will no longer have this variable set?